applications/launchpad/gui-react/src/containers/WalletContainer/Send/styles.ts

```diff
@@ -21,8 +22,11 @@ export const TagWrapper = styled.div`
 
 export const FormButtons = styled.div`
   display: flex;
+  position: relative;
   justify-content: space-between;
+  padding: ${({ theme }) => `${theme.spacing(0.67)} ${theme.spacing(1.67)}`};
   column-gap: ${({ theme }) => theme.spacingHorizontal(1)};
+  border-top: 1px solid ${({ theme }) => theme.borderColor};
```

Comment:
this gets lost sometimes on Dark Mode, `theme.selectBorderColor` could be another option if needed.

Reply:
You are right! Going to change this